hypervisor/boot/cmdline.c

```diff
@@ -24,15 +27,15 @@ int32_t parse_hv_cmdline(void)
 		return -EINVAL;
 	}
 
-	mbi = (struct multiboot_info *)(hpa2hva((uint64_t)boot_regs[1]));
+	mbi = (struct multiboot_info *)((void *)((uint64_t)boot_regs[1]));
```

Comment:
How can we remove hpa2hva?

Reply:
Will add hpa2hva_early and use hpa2hva_early instead